content/plugins/datamints/kirby-plugin_ldap/plugin.txt

```diff
@@ -0,0 +1,13 @@
+Title: Datamints Kirby-Plugin Ldap
```

Comment:
Your developer name will be automatically printed below the plugin name. And that it's a Kirby plugin is obvious as the plugin will be listed in our plugin directory.

Suggestion:

```suggestion
Title: LDAP login
```